docs/reference/rest-api/security/update-settings.asciidoc

```diff
@@ -18,27 +18,28 @@ Updates the settings of the security internal indices.
 ==== {api-request-body-title}
 
 `security`::
-(Optional, object) Settings to be used for the index used for most security 
+(Optional, object) Settings to be used for the index used for most security
 configuration, including Native realm users and roles configured via the API.
 
 `security-tokens`::
-(Optional, object) Settings to be used for the index used to store 
+(Optional, object) Settings to be used for the index used to store
 <<security-api-get-token,tokens>>.
 
-`security`::
-(Optional, object) Settings to be used for the index used to store 
+`security-profile`::
+(Optional, object) Settings to be used for the index used to store
 <<security-api-activate-user-profile, profile>> information.
 
 
 [[security-api-update-settings-desc]]
 ==== {api-description-title}
-This API allows a user to modify the settings for the Security internal indices 
-(`.security` and associated indices). Only a subset of settings are allowed to 
+This API allows a user to modify the settings for the Security internal indices
+(`.security` and associated indices). Only a subset of settings are allowed to
 be modified. This includes:
 
 - `index.auto_expand_replicas`
 - `index.number_of_replicas`
 
+NOTE: If `index.auto_expand_replicas` is set, `index.number_of_replicas` will be ignored during updates.
```

Comment:
This is change 2: I _think_ it's as simple as that. Basically, if you have e.g., 0-1 for auto expand, and a 1 node cluster, trying to set number of replicates to say 2 won't have any effect (https://github.com/elastic/elasticsearch/issues/27871)

Reply:
Matches my understanding, and this API just plumbs through to the normal settings infra so it should behave the same as the same setting on any other index.